.github/workflows/push_docker.yaml

```diff
@@ -31,33 +40,32 @@ jobs:
           sudo rm -rf "/usr/local/share/boost"
           sudo rm -rf "$AGENT_TOOLSDIRECTORY"
       - name: Checkout repository
-        uses: actions/checkout@v3
-      - name: Docker Hub login
-        uses: docker/login-action@v2
-        with:
-          username: ${{ secrets.DOCKER_HUB_USERNAME }}
-          password: ${{ secrets.DOCKER_HUB_TOKEN }}
```

Comment:
The `master` branch's workflow file would need this, one way or another. In general it seems like this PR is not ready to be merged to `master`. Please update the workflow file so that *it is as it should be on the `master` branch* after merging.